lib/dts-lib.robot

```diff
@@ -92,22 +92,22 @@ Run EC Transition
     Sleep    10s
 
 Flash Firmware In DTS
-    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
-    ...    ability for flashing firmware work correctly.
-    [Arguments]    ${fw_dl_link}=${FW_DOWNLOAD_LINK}
-    Execute Command In Terminal
-    ...    wget -O /tmp/coreboot.rom ${fw_dl_link}
-    ${out}=    Execute Command In Terminal
-    ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
-    ...    timeout=320s
-    ${verified}=    Run Keyword And Return Status
-    ...    Should Contain    ${out}    VERIFIED
-    IF    ${verified} == ${FALSE}
-        ${out}=    Execute Command In Terminal
-        ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
-        ...    timeout=320s
-        Should Contain    ${out}    identical
+    [Arguments]    ${fw_file_path}    ${region}=${EMPTY}
```

Comment:
@wiktormowinski Do I see correctly, or you simply copied over the body of the `Flash Via Internal Programmer` kwd from the `lib/flash.robot`?

This does not sound great.

You should probably remove `Flash Firmware in DTS` from here, and replace it's occurrences with the one from `lib/flash.robot`.

Reply:
that is excatly what I did, and now it is fixed